src/layouts/login/LoginLayout.astro

```diff
@@ -0,0 +1,122 @@
+---
+import BaseLayout from "../BaseLayout.astro";
+import type {
+  ExtendedAssignmentData,
+  ExtendedProjectData,
+  MyProfile,
+  LoginMethod,
+} from "../../Types";
+import config from "../../config.json";
+import { LoginMethodSelector } from "@apps/auth-login/LoginMethodSelector";
+
+export interface Props {
+  title: string;
+  i18n: Node;
```

Comment:
I'm getting an error in `sign-in.astro` because of this. i18n isn't a Node, but of Type `Translations`.

Reply:
Fixed